subprojects/distributions-integ-tests/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -57,7 +57,7 @@ abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
      * Change this whenever you add or remove subprojects for distribution-packaged plugins (lib/plugins).
      */
     int getPackagedPluginsJarCount() {
-        46
+        45
```

Comment:
Which dependency has been removed here? Is removing the dependency a breaking change?

Reply:
It's `apiguardian-api.jar`. I guess it must have been dropped from JUnit?

Reply:
Yeah, it used to be a dependency of junitplatform 1.7.2:

```text
|    |    |    +--- project :testing-junit-platform
|    |    |    |    +--- project :base-services (*)
|    |    |    |    +--- project :messaging (*)
|    |    |    |    +--- project :platform-jvm (*)
|    |    |    |    +--- project :language-java (*)
|    |    |    |    +--- project :testing-base (*)
|    |    |    |    +--- project :testing-jvm (*)
|    |    |    |    \--- org.junit.platform:junit-platform-launcher -> 1.7.2
|    |    |    |         +--- org.junit:junit-bom:5.7.2
|    |    |    |         |    +--- org.junit.platform:junit-platform-engine:1.7.2 (c)
|    |    |    |         |    +--- org.junit.platform:junit-platform-launcher:1.7.2 (c)
|    |    |    |         |    \--- org.junit.platform:junit-platform-commons:1.7.2 (c)
|    |    |    |         +--- org.apiguardian:apiguardian-api:1.1.0
|    |    |    |         \--- org.junit.platform:junit-platform-engine:1.7.2
|    |    |    |              +--- org.junit:junit-bom:5.7.2 (*)
|    |    |    |              +--- org.apiguardian:apiguardian-api:1.1.0
|    |    |    |              +--- org.opentest4j:opentest4j:1.2.0
|    |    |    |              \--- org.junit.platform:junit-platform-commons:1.7.2
|    |    |    |                   +--- org.junit:junit-bom:5.7.2 (*)
|    |    |    |                   \--- org.apiguardian:apiguardian-api:1.1.0
```

https://ge.gradle.org/s/soxzctqhtkyyg/dependencies?dependencies=guar&expandAll&focusedDependency=WzI2LDAsMjI0NSxbMCwwLFswXV1d&focusedDependencyView=versions